### PR TITLE
 disable the selinux before build the image

### PR DESCRIPTION
**Pull request by wanghaoran1988**
fix  https://github.com/caicloud/cyclone/issues/53

### CODE REVIEW

File: scripts/build-image.sh [hunk 1]
```diff
@@ -24,7 +24,13 @@ function usage {
   echo -e "Environment variable:"
   echo -e " PUSH_TO_REGISTRY     \tPush images to caicloud registry or not, options: Y or N. Default value: ${PUSH_TO_REGISTRY}"
 }
-
+function disable-selinux() {
+  if selinuxenabled && [ "$(getenforce)" = "Enforcing" ]; then
```
Comment (gaocegege) [inline scripts/build-image.sh]: I run your PR in OS X 10.12.1, and it raises an error:

```
./scripts/build-image.sh: line 28: selinuxenabled: command not found
```

maybe we should define it before this function :)

File: scripts/build-image.sh [hunk 2]
```diff
@@ -65,10 +71,11 @@ echo "+++++ Start building cyclone server"
 cd ${ROOT}
 
 # Build cyclone comtainer.
+disable-selinux
```
Comment (gaocegege) [inline scripts/build-image.sh]: By the way, could you add some comments on it :)

File: scripts/build-image.sh (hunk 1 above)
Comment (wanghaoran1988) [inline scripts/build-image.sh]: @gaocegege Sorry, don't have an mac on hand, I am using fedora, will have an update, when a mac in hand.

Comment (gaocegege) [inline scripts/build-image.sh]: No worry :) 
Is `selinuxenabled ` a built-in command in fedora?

Comment (wanghaoran1988) [inline scripts/build-image.sh]: selinuxenabled - tool to be used within shell scripts to determine if selinux is enabled

Comment (superxi911) [inline scripts/build-image.sh]: +1
> OK, I think, we should bypass this function call when the script is running in OS X and other OSes which don't have SELinux features :)

Comment (wanghaoran1988) [inline scripts/build-image.sh]: ok

**Comment by gaocegege** [comment]
LGTM, it works for me. PTAL @superxi911 
😄

**Comment by superxi911** [comment]
LGTM

**Pull request merged**